generate_analysis_graphs.py

```diff
@@ -112,3 +112,32 @@
         )
         chart.save(f"{output_dir}/season_distribution_{plan.analysis_file_key}.html")
         chart.save(f"{output_dir}/season_distribution_{plan.analysis_file_key}.png", scale_factor=IMG_SCALE_FACTOR)
+
+    # Compute the number of UIDs with manually labelled relevant messages per show
+    log.info("Graphing the No. of UIDs with relevant messages per show...")
+    relevant_uids_per_show = OrderedDict()
+
+    for plan in PipelineConfiguration.RQA_CODING_PLANS:
+        relevant_uids_per_show[plan.raw_field] = 0
+    for msg in messages:
+        for plan in PipelineConfiguration.RQA_CODING_PLANS:
+            if plan.binary_code_scheme is not None and msg["consent_withdrawn"] == "false":
```

Comment:
This might be easier to read if you said:
```
if msg["consent_withdrawn"] == Codes.TRUE:
    continue
```
first.

Then you can:
```
if plan.binary_code_scheme is None:
    ...
else:
    ...
```